homeassistant/components/tuya/select.py

```diff
@@ -307,6 +307,35 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Moes Smart Control Panel Mini
+    # Not documented
+    "dgnzk": (
+        SelectEntityDescription(
+            key=DPCode.LIGHT_MODE,
+            entity_category=EntityCategory.CONFIG,
+            translation_key="light_mode"
+        ),
+        SelectEntityDescription(
+            key=DPCode.RELAY_STATUS,
+            entity_category=EntityCategory.CONFIG,
+            translation_key="overall_relay_status"
```

Comment:
since these are used as to define state/behavior and not to report status, maybe rename to `overall_relay_behavior` (which also matches their strings)? Also for the others: `relay_behavior_X`

Reply:
Sorry I don't get what you mean? The entity gets and sets the state of the relay. I don't think your change would be appropriate

Reply:
For the `select` entities here I was basing my suggestion on the fact that the strings file has a `memory` as part of the list of possible values, and also it is named as "Overall relay behavior". So it looks to me that it gets/sets that startup behavior of the relays, and not the current state. But I may be misunderstanding it.

Aren't the `switch` entities the ones that set the state of the relays?

Reply:
Switch entities do set the state however you are assuming that they are the only source of truth. This would not be the case as Tuya reports the state to HA. So the switch entities in this case show the current state as well as setting a new state

Reply:
But my suggestion was for the `select` entities, not for the `switch` entities.